prow/plugins/config.go

```diff
@@ -1332,6 +1333,12 @@ func (c *Configuration) Validate() error {
 		return err
 	}
 
+	for _, milestone := range c.RepoMilestone {
+		if milestone.MaintainersID != 0 {
+			logrus.Warnf("deprecated field: maintainers_id is configured for repo_milestone, maintainers_team should be used instead")
```

Comment:
We reload the config and don't want to show this all the time, please use something like this: https://github.com/kubernetes/test-infra/blob/9703a3111217af8a9a4616883ab316c28f33e3cf/prow/plugins/config.go#L911

Reply:
Ah, that makes sense!